backend/src/schema/index.js

```diff
@@ -35,14 +35,14 @@ export default applyScalars(
             'Notfication',
             'Post',
             'Comment',
-            'Report',
             'Statistics',
             'LoggedInUser',
             'Location',
             'SocialMedia',
             'User',
             'EMOTED',
             'NOTIFIED',
+            'REPORTED',
```

Comment:
I guess that you want to exclude the `REPORTED` query as well, otherwise we have one for `reports` and another for `REPORTED`

Reply:
Yes, genau!

Reply:
sorry, you marked as resolved, but didn't exclude `REPORTED` for queries... maybe by accident?

Reply:
I did locally not yet pushed up.
You mean to exclude it from the queries list, I suppose?